README.md

```diff
@@ -12,34 +12,28 @@ A simple skeleton application to get you started on React/TypeScript development
 1. Copy or make a fork of this repository (you can delete `.policy.yml`)
 1. Run `npm install`
 1. Create an application via the Foundry Developer Console with the configuration below. Once the application has been created, come back to these instructions.
-    1. Select client facing application (creates a public client) when asked what type of application you're building
-    1. Add `https://localhost:8080/auth/callback` as a redirect URL when asked for one
-    1. If you know where your application is getting deployed, you can add your production URL as well: `https://example.app.com/auth/callback`
+    1. Add `http://localhost:8080` to your CORS policy in Control Panel, see [Configure CORS](https://www.palantir.com/docs/foundry/administration/configure-cors/index.html) for more details.
+    2. Select client facing application (creates a public client) when asked what type of application you're building
+    3. Add `http://localhost:8080/auth/callback` as a redirect URL when asked for one
+    4. If you know where your application is getting deployed, you can add your production URL as well: `https://example.app.com/auth/callback`
 1. Follow the instructions in the Foundry Developer Console to generate and then install your application's Ontology SDK via the application-specific NPM registry
 1. Run `mv .env.development.sample .env.development` to start setting up your environment variables, and fill in the fields marked with `<>`
 
-    a. You can find your Client ID in either the Quickstart guide for the SDK or in the Permissions & OAuth page
+    Note! You can find your Client ID in the Permissions & OAuth page or in the Getting started guid under the API documentation
 
 1. Update the following files with your Ontology SDK and Object types:
     1. Update [`src/utils/client.ts`](./src/utils/client.ts) with the correct package name
-    1. Update [`src/pages/home.tsx`](./src/pages/home.tsx) with the correct package name and Ontology Objects you want to use.
+    2. Update [`src/pages/home.tsx`](./src/pages/home.tsx) with the correct package name and Ontology Objects you want to use.
```

Comment:
github automaticaly numbers these so we don't have to keep them up to date manually
```suggestion
    1. Update [`src/pages/home.tsx`](./src/pages/home.tsx) with the correct package name and Ontology Objects you want to use.
```